### ISSUE TITLE
Radio Group should be wrapped in a fieldset

### ISSUE THREAD

**Issue by coliff** [trimble-oss/modus-web-components#2487]
Radio groups would benefit if they wrapped in a fieldset so you can tab once into it and then use arrow keys to go up and down.

Demo and more info: https://www.w3.org/WAI/tutorials/forms/grouping/
